docs/src/maintainer-guide/npm-2fa.md

```diff
@@ -0,0 +1,21 @@
+---
+title: npm two-factor authentication
+layout: doc
+edit_link: https://github.com/eslint/eslint/edit/master/docs/maintainer-guide/npm-2fa.md
```

Comment:
```suggestion
edit_link: https://github.com/eslint/eslint/edit/main/docs/maintainer-guide/npm-2fa.md
```

Note to self: this doc is out of date. I can update it after this PR goes in.

Reply:
This file was actually removed in https://github.com/eslint/eslint/pull/13622, it currently does not exist in [docs/src/maintainer-guide](https://github.com/eslint/eslint/tree/main/docs/src/maintainer-guide). I'd guess that the automation that produced this change mistakenly pulled this file back from the website repo.

Reply:
Ah that makes sense! Yeah, we can remove this.